runtime/admin/src/main/java/org/apache/polaris/admintool/BootstrapCommand.java

```diff
@@ -80,6 +80,12 @@ static class FileInputOptions {
     }
 
     static class SchemaInputOptions {
+      @CommandLine.Option(
```

Comment:
The schema init scripts _must_ be idempotent, because the server runtime executes them whenever it loads a new realm. So why do we need to specify this here? Couldn't we just assume that schema should be set up always? If we want to avoid re-running init scripts on an already-initialized database, this is a concern that affects both the admin tool and the server. IOW this should be controlled by a configuration option in application.properties.